include/tvm/tir/schedule/schedule.h

```diff
@@ -305,6 +305,42 @@ class ScheduleNode : public runtime::Object {
   virtual BlockRV CacheWrite(const BlockRV& block_rv, int write_buffer_index,
                              const String& storage_scope) = 0;
   /******** Schedule: Compute location ********/
+  /*!
+   * \brief Move a producer block under the specific loop, and regenerate the loops induced by the
+   * block so that the buffer region generated by the producer block could cover those regions
+   * written by the producers. It requires:
+   * 1) `block` and `loop` are under the same scope, `loop` is not the ancestor of `block`
+   * 2) The scope block has stage-pipeline property
+   * 3) The subtree of the scope block, where the given block is in, satisfies the compact dataflow
+   * condition. i.e. all the blocks in the scope block's subtree must be either complete block or
+   * reduction block
+   * 4) The block is not an output block, i.e. the buffer regions written by the block are allocated
+   * under the current scope
```

Comment:
I want to make it more clear.
```suggestion
   * 4) The block is not an output block with regard to the scope block, i.e. the buffers written by the block are allocated
   * under the scope block
```

Reply:
Yeah this makes sense to me